js/ajaxTask.js

```diff
@@ -0,0 +1,34 @@
+function send() {
+    let username = getElementByIdFromDocument("username").value;
+    let password = getElementByIdFromDocument("password").value;
+    let confirmPassword = getElementByIdFromDocument("confirmPassword").value;
+
+    let object = { username, password, confirmPassword };
+    let jsonToBeSent = converObjectToJSON(object);
+
+    const url = "./register.php"
+    let callback = function (text) {
+        console.log(text);
+    };
+
+    ajax(url, { success: callback, data: jsonToBeSent });
+}
+function getElementByIdFromDocument(elementName) {
+    return document.getElementById(elementName);
+}
+function converObjectToJSON(object) {
```

Comment:
Typo, in case you want to keep the function:
```suggestion
function convertObjectToJSON(object) {
```